blockchain/message_pool/src/msgpool.rs

```diff
@@ -57,15 +55,18 @@ impl MsgSet {
                 let gas_price = exms.message().gas_price();
                 let rbf_num = BigUint::from(RBF_NUM);
                 let rbf_denom = BigUint::from(RBF_DENOM);
-                let min_price =
-                    gas_price.clone() + (gas_price / &rbf_num) + rbf_denom + BigUint::from(1_u64);
+                let min_price = gas_price.clone() + ((gas_price * &rbf_num) / rbf_denom) + 1u8;
                 if m.message().gas_price() <= &min_price {
                     // message with duplicate sequence is already in mpool
+                    warn!("try to add message with duplicate sequence");
                     return Err(Error::DuplicateSequence);
```

Comment:
This is the wrong error message, gas price being less has nothing to do with duplicate sequence